internal/types/host/keepalived_host_collector_host.go

```diff
@@ -97,6 +104,22 @@ func (k *KeepalivedHostCollectorHost) getKeepalivedVersion() (*version.Version,
 	return utils.ParseVersion(stderr.String())
 }
 
+func isEnableJsonSupported() (bool, error) {
+        cmd := exec.Command("keepalived", "--version")
+        output, err := cmd.CombinedOutput()
+        if err != nil {
+                return false, fmt.Errorf("failed to execute keepalived --version: %v", err)
+        }
+
+        outputStr := string(output)
+
+        if strings.Contains(outputStr, "--enable-json") {
+                return true, nil
+        }
+
+        return false, fmt.Errorf("keepalived does not turn on the enable-json switch")
```

Comment:
If you're gonna return an error when it's not available, you don't need the extra `supported` argument to return.
(Actually, you haven't used the argument where you called this function.

Reply:
Sorry, I originally wanted to add a metric to show the status of whether keepalived enable-json is supported, controlled by this supported return value. Later I thought it was a bit redundant and forgot to delete it.
I'll modify it later.